frontend/app/assets/javascripts/largetree_dragdrop.js.erb

```diff
@@ -9,6 +9,7 @@
         this.dragActive = false;
         this.dragIndicator = $('<div class="tree-drag-indicator" />');
         this.rowsToMove = [];
+        this.dropBehavior = undefined;
```

Comment:
Can this just be initialized to `"before"` to match the default in the toolbar?

Reply:
Basically; had to also consider local storage.